force-app/main/default/lwc/scopedNotification/scopedNotification.html

```diff
@@ -17,7 +17,7 @@
                 variant={iconVariant}
             ></lightning-icon>
         </div>
-        <div class="slds-media__body" style="white-space: pre">
+        <div class="slds-media__body">
```

Comment:
@bethbrains - why did we need to remove the white-space here? On page 1 we have multiple errors and removing that style no longer shows each error on a single line.